homeassistant/components/plugwise/sensor.py

```diff
@@ -430,18 +433,20 @@ async def async_setup_entry(
 class PlugwiseSensorEntity(PlugwiseEntity, SensorEntity):
     """Represent Plugwise Sensors."""
 
+    entity_description: PlugwiseSensorEntityDescription
+
     def __init__(
         self,
         coordinator: PlugwiseDataUpdateCoordinator,
         device_id: str,
-        description: SensorEntityDescription,
+        description: PlugwiseSensorEntityDescription,
     ) -> None:
         """Initialise the sensor."""
         super().__init__(coordinator, device_id)
         self.entity_description = description
         self._attr_unique_id = f"{device_id}-{description.key}"
 
     @property
-    def native_value(self) -> int | float | None:
+    def native_value(self) -> int | float:
```

Comment:
It seems odd / wrong that none of these sensors cannot ever become in a state the value is unknown? Is this a bug?

Reply:
In my mind this is not a bug: there is always xml-data to collect from the Plugwise device, the data might be outdated but it is always there. Only when the connection to the Plugwise device is lost, then no new data (for all sensors, etc.) is coming in, but that situation is handled by the DUC.
With that in mind, I thought let's get rid of the `None`.

Reply:
> In my mind this is not a bug: there is always xml-data to collect from the Plugwise device, the data might be outdated but it is always there.

Right, although odd a bit from a device perspective, it makes sense. 👍 

../Frenck